<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThumbnailList</class>
 <widget class="QDialog" name="ThumbnailList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Georeferenced thumbnails</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/sumbandilasat/icons/general/list.png</normaloff>:/sumbandilasat/icons/general/list.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QListWidget" name="listWidget">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionRemove">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/georeferencer/remove.png</normaloff>:/icons/georeferencer/remove.png</iconset>
   </property>
   <property name="text">
    <string>Remove thumbnail</string>
   </property>
   <property name="toolTip">
    <string>Remove a thumbnail</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
